app/models/project.rb

```diff
@@ -33,4 +37,15 @@ def related_projects
   def tasks_url
     "#{github_url}/issues"
   end
+
+  def self.parse_git_url(url)
+    url.gsub(/^(((https|http|git)?:\/\/(www\.)?)|git@)github.com(:|\/)/i, '').gsub(/(\.git|\/)$/i, '')
```

Comment:
Line is too long. [102/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.